docs/arcaflow_scenarios/memory_hog.md

```diff
@@ -0,0 +1,13 @@
+# Memory Hog
+This scenario is based on the arcaflow [arcaflow-plugin-stressng](https://github.com/arcalot/arcaflow-plugin-stressng) plugin. 
+The purpose of this scenario is to create Virtual Memory pressure on a particular node of the Kubernetes/OpenShift cluster for a time span.
+To enable this plugin add the pointer to the scenario input file `scenarios/arcaflow/memory-hog/input.yaml` as described in the 
+Usage section.
+This scenario takes the following input parameters:
+
+- **kubeconfig :** *string* the kubeconfig needed by the deployer to deploy the sysbench plugin in the target cluster
+**Note:** this parameter will be automatically filled by kraken if the `kubeconfig_path` property is correctly set
+- **node_selector :** *key-value map* the node label that will be used as `nodeSelector` by the pod to target a specific cluster node
```

Comment:
Can this be a string to be flexible? Thoughts?

Reply:
unfortunately that's not possible due to the kubernetes deployer interface, it must be a map.